frontend/assessments/1/style.scss

```diff
@@ -0,0 +1,116 @@
+body {
+    background-color: #fba4a6;
+    font-family:Poppins;
+}
+
+nav {
+    display: flex;
+    justify-content: space-around;
+    padding: 5px;
+    margin: 10px;
+
+    a{
+        text-decoration: none;
+        padding: 20px;
+        color: white;
+        font-size: 1.3rem;
+    }
+}
+
+.container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    color: white;
+    width: 100%;
+    height: 90vh;
+    margin: 0 10px;
+
+    h1 {
+        // font-size: 5rem;
+        margin: 10px;
+    }
+
+    p {
+        // font-size: 1.5rem;
+        margin: 5px;
+    }
+    button {
+        padding: 15px 20px;
+        margin-top: 30px;
+        font-size: 1.2rem;
+        border-radius: 10px;
+        background-color: #be5569;
```

Comment:
Lets have a variables file for these values, and colors. Also use rem/em whenever possible.